Corona-Warn-App/src/main/java/de/rki/coronawarnapp/worker/BackgroundConstants.kt

```diff
@@ -39,6 +39,13 @@ object BackgroundConstants {
      */
     const val POLLING_VALIDITY_MAX_DAYS = 21
 
+    /**
+     * The time when rat polling is switched to a larger interval
+     *
+     * @see TimeUnit.MINUTES
+     */
+    const val RAT_POLLING_END_OF_PHASE1_MINUTES = 90
```

Comment:
Can we move this to where it is accessed? I don't think it is reused somewhere.

Reply:
done!